src/kOS/AddOns/ActionGroupsExtended/ActionGroupsExtendedAPI.cs

```diff
@@ -26,6 +26,10 @@ public ActionGroupsExtendedAPI()
         /// <returns>Is AGX installed?</returns>
         public bool Installed() 
         {
+            if (calledType == null)
+            {
+                return false;
+            }
             try //try-catch is required as the below code returns a NullRef if AGX is not present.
```

Comment:
That seems fine to me.  Just remove the comment because we don't know that it's true anymore.